Documentation/HTKToMRTKPortingGuide.md

```diff
@@ -125,7 +129,7 @@ Some Utilities have been reconciled as duplicates with the Solver system. Please
 
 | HTK 2017 |  MRTK v2  |
 |----------|-----------|
-| Billboard | [`RadialView`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.RadialView) [Solver](README_Solver.md) |
+| Billboard | [`Billboard`](xref:Microsoft.MixedReality.Toolkit.UI.Billboard) (Need to add new component with new namespace.) |
```

Comment:
(Need to add new component with new namespace.)

Is that a TODO for us, or is that an instruction to the user? I'm not really sure what to do with that text.

Reply:
Good feedback, I was trying to communicate that developers still need to take action during port. Even though it's Billboard and Billboard, they are not the same.